sphinxgallery/tests/test_get_rst.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Author: Óscar Nájera
+# License: 3-clause BSD
+"""
+Testing the rst files generator
+"""
+from __future__ import division, absolute_import, print_function
+import sphinxgallery.gen_rst as sg
+from nose.tools import assert_equals
+
+
+def test_split_code_and_text_blocks():
+    """Test if a known example file gets properly split"""
+
```

Comment:
It would be quite nice to include a more complicated example. You can create the file yourself with `with tempfile.NamedTemporaryFile():`

What I have in mind is more than two blocks testing a few edge cases like an inline string

```
"""This should not be part of a 'text' block
```
a function docstring
```python
def dummy():
    """This should not be part of a 'text' block'
```

a comment that is separated from a splitter by a new line something like:

```
######################################
# asdf

# this should not be part of a 'text' block
```

and possibly other edge cases that you think about.

Testing the content as well as the type of the block would be great too.